### PR TITLE
Add ignore-2.12.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Please, after merge, add 2.11 to the test matrix (I didn't find it)

Depends-On: https://github.com/ansible-collections/cisco.ios/issues/300

https://github.com/ansible-collections/overview/issues/45#issuecomment-813821100

### CODE REVIEW

**Comment by NilashishC** [comment]
@Andersson007 Thank you for this PR! A trivial changelog fragment is needed for CI to pass. Could you please add that too?

**Comment by Andersson007** [comment]
@NilashishC done, thanks

File: changelogs/fragments/296-add_ignore_txt.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+trivial:
```
Comment (Andersson007) [inline changelogs/fragments/296-add_ignore_txt.yml:1]:
```suggestion
---
trivial:
```

File: changelogs/fragments/296-add_ignore_txt.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+- Add ingore-2.12.txt.
```
Comment (Andersson007) [inline changelogs/fragments/296-add_ignore_txt.yml:3]:
```suggestion
  - Add ingore-2.12.txt.
```

**Comment by Andersson007** [comment]
the following errors are unrelated
```
ERROR: Found 9 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/ios_bgp_address_family.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> neighbor
ERROR: plugins/modules/ios_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor
ERROR: plugins/modules/ios_ntp.py:0:0: no-log-needed: Argument 'auth_key' in argument_spec could be a secret, though doesn't have `no_log` set
ERROR: plugins/modules/ios_ospf_interfaces.py:0:0: no-log-needed: Argument 'key_chain' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication
ERROR: plugins/modules/ios_ospfv3.py:0:0: no-log-needed: Argument 'key_chain' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
ERROR: plugins/modules/ios_ospfv3.py:0:0: no-log-needed: Argument 'key_chain' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> sham_link -> authentication
ERROR: plugins/modules/ios_ospfv3.py:0:0: no-log-needed: Argument 'key_chain' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
ERROR: plugins/modules/ios_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set
ERROR: plugins/modules/ios_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set found in aggregate
```
so ready for review

**Comment by Andersson007** [comment]
Depends-On: https://github.com/ansible-collections/cisco.ios/issues/300

**Comment by Andersson007** [comment]
recheck

**Review by NilashishC — APPROVED** [review]

**Comment by Andersson007** [comment]
@NilashishC Depends-On doesn't work with issues, right?

**Comment by justjais** [comment]
Closing this PR as have added the fix to PR #304 as both were interdependent for tests to turn green.

cc. @Andersson007

**Comment by Andersson007** [comment]
@justjais cool, thanks!

**Pull request closed without merging**